src/components/ebay-textbox/index.js

```diff
@@ -0,0 +1,31 @@
+const markoWidgets = require('marko-widgets');
+const processHtmlAttributes = require('../../common/html-attributes');
+const template = require('./template.marko');
+
+function getInitialState(input) {
+    const htmlAttributes = input.invalid ?
+        Object.assign({}, processHtmlAttributes(input), { 'aria-invalid': 'true' }) : processHtmlAttributes(input);
```

Comment:
IMO The user's `HtmlAttributes` should be left alone and sent to the template from `processHtmlAttributes(input)`. You can position your own HTML attributes (`aria-invalid`) before theirs, so that ours take precedence. I imagine it would be very uncommon to have the user attempt to set this, but if are worried about that, we would need to handle it across all components.

Reply:
@dasarianudeep I believe you should address this. Let us know if you have questions.